### PR TITLE
Simplify the selection of the dynamic port in the nodejs sdk.

### PR DESCRIPTION
**Pull request by roberthbailey**
This brings the NodeJS SDK in line with the C++ / Unity / Rust SDKs.

Part of #851.

### CODE REVIEW

File: sdks/nodejs/src/agonesSDK.js [hunk 1]
```diff
@@ -25,22 +25,8 @@ class AgonesSDK {
 	}
 
 	get port() {
-		const defaultPort = '59357';
 		const port = process.env.AGONES_SDK_GRPC_PORT;
```
Comment (steven-supersolid) [inline sdks/nodejs/src/agonesSDK.js]: In JavaScript it's common to use `||` e.g. do `return process.env.AGONES_SDK_GRPC_PORT || '59357'`
Without examining grpc code too much it seems port should be a number so you could cast using `return Number(process.env.AGONES_SDK_GRPC_PORT) || 59357`

Comment (steven-supersolid) [reply sdks/nodejs/src/agonesSDK.js]: doh too slow :)

Comment (roberthbailey) [reply sdks/nodejs/src/agonesSDK.js]: I'm happy to fix it in a follow-up PR. :)

Comment (roberthbailey) [reply sdks/nodejs/src/agonesSDK.js]: https://github.com/googleforgames/agones/pull/1129

> it seems port should be a number

The value is returned and concatenated with 'localhost' so I think returning a string is ok.

**Pull request merged**